contracts/fungible-token.aes

```diff
@@ -57,16 +63,20 @@ contract FungibleToken =
   public function allowance_from(addr: address) : int =
     Map.lookup_default((addr, Call.caller), state.allowances, 0)
 
+  // TODO check negative allowances possible
   public stateful function transfer_from(from: address, to: address, value: int) =
     require(allowance_from(from) >= value, "The provided value is larger than what is allowances.")
+    // extract calculations froms state updates
     put(state{ allowances[(from, Call.caller)] = allowance_from(from) - value })
     internal_transfer(from, to, value)
     Chain.event(Transfer(from, to, value))
 
+  // TODO check failure if allowance doesn't exist yet
```

Comment:
If the allowance doesn't exist yet it should be assumed that the value is 0, and yes this should be added in the Sophia implementation, thanks :)